analyses/org.osate.analysis.resource.budgets/plugin.xml

```diff
@@ -265,13 +265,20 @@ censes only apply to the Third Party Software and not any other portion of this
          </command>
       </menuContribution>
    </extension>
-   <extension
-         id="toc_busload"
-         name="Bus Load Analysis Help Table of Contents"
-         point="org.eclipse.help.toc">
-      <toc
-            file="help/toc_busload.xml"
-            primary="true">
-      </toc>
+   <extension point="org.eclipse.emf.ecore.generated_package">
+      <!-- @generated analysis -->
+      <package
+            uri="http://osate.org/analysis/analysis"
+            class="org.osate.analysis.resources.budgets.internal.models.analysis.AnalysisPackage"
+            genModel="model/analysis.genmodel"/>
+   </extension>
```

Comment:
Looks like this should be removed.

Reply:
I thought I fixed this the first time, but someone I didn't.  Put this back they way it should.  Was able to see the help text in my product build.

Reply:
This generate_package extension still needs to be removed. The referenced class doesn't exist, I guess this is left over from before you created the new plugin.

Reply:
Urgh.  Got committed, but not pushed this time.  I don't know what's going on with me today.  Pushed out for real this time.

Reply:
It disappeared again. Every time I regenerate the busload.ecore classes it overwrites the help text entry in the plugin.xml file.

Put it back again...